src/app.js

```diff
@@ -1 +1,43 @@
 'use strict';
+
+const { createNumber } = require('./createNumber');
+const { calculateBullsCows } = require('./bullsAndCows');
+const { myInterface } = require('./myInterface');
+
+const gameNumberLength = 4;
+
+const game = () => {
+  const winNumber = createNumber(gameNumberLength);
```

Comment:
And also we don't need to make it with different amount of numbers, we should work only with 4 numbers